src/types/generatedGQL.ts

```diff
@@ -25,6 +25,39 @@ export interface Conditions {
 // @generated
 // This file was automatically generated and should not be edited.
 
+// ====================================================
```

Comment:
Please run prettier in this file

Reply:
Maybe we can add some kind of husky hook , so we are sure that prettier was executed

Reply:
We will ignore this file in the eslint/prettier configuration